superset/db_engine_specs/mysql.py

```diff
@@ -53,7 +53,7 @@
 CONNECTION_UNKNOWN_DATABASE_REGEX = re.compile("Unknown database '(?P<database>.*?)'")
 
 
-class MySQLEngineSpec(BaseEngineSpec):
+class MySQLBaseEngineSpec(BaseEngineSpec):
```

Comment:
There's no need to split the class into two. Just add the mixin and attributes here.

(Postgres was already split into two, probably because of Redshift.)